message/validation/message_counts.go

```diff
@@ -0,0 +1,56 @@
+package validation
+
+import (
+	specqbft "github.com/bloxapp/ssv-spec/qbft"
+	spectypes "github.com/bloxapp/ssv-spec/types"
+
+	"github.com/bloxapp/ssv/protocol/v2/ssv/queue"
+)
+
+// maxMessageCounts is the maximum number of acceptable messages from a signer within a slot & round.
+func maxMessageCounts(committeeSize int) MessageCounts {
+	return MessageCounts{
+		PreConsensus:  1,
+		Proposals:     1,
+		Prepares:      1,
+		Commits:       committeeSize + 1,
```

Comment:
@nkryuchkov need to add the decide messages. 
based on @MatheusFranco99 calculation